test/f2fs.config

```diff
@@ -0,0 +1,6 @@
+image test.f2fs {
+	f2fs {
+		label = "f2fstest"
+	}
+	size = 64M
+}
```

Comment:
Please add a test for this to `test/basic-images.test`. It should probably look something like this:
```
exec_test_set_prereq mkfs.f2fs
exec_test_set_prereq sload.f2fs
test_expect_success mkfs_f2fs,sload_f2fs "f2fs" "
	run_genimage f2fs.config test.f2fs
	...
"
```
And do some testing here. Probably fsck.f2fs and compare the output of dump.f2fs to an existing file or something like that. Just some basic checks that the filesystem is valid and actually contains the expected files.
Take a look at `check_ext` for some inspiration.